tsconfig.angular.json

```diff
@@ -36,6 +36,7 @@
         "enableI18nLegacyMessageIdFormat": false,
         "strictInjectionParameters": true,
         "strictInputAccessModifiers": true,
-        "strictTemplates": true
+        "strictTemplates": true,
+        "types": ["node", "jasmine"]
```

Comment:
Is this necessary for the entire project or only for the tests?
If it is only necessary for the tests, then it should be done in tsconfig.spec.json.